app/src/test/java/org/mozilla/focus/searchsuggestions/SearchSuggestionsViewModelTest.kt

```diff
@@ -0,0 +1,33 @@
+package org.mozilla.focus.searchsuggestions
+
+import android.arch.lifecycle.Lifecycle
+import android.arch.lifecycle.LifecycleOwner
+import android.arch.lifecycle.LifecycleRegistry
+import org.junit.Test
+
+import org.junit.Assert.*
+import org.junit.Rule
+import org.mockito.Mockito.*
+import android.arch.core.executor.testing.InstantTaskExecutorRule
+import org.junit.rules.TestRule
+
+
```

Comment:
\[ktlint\] WARNING: \[no\-consecutive\-blank\-lines\] Needless blank line\(s\) in column 1